doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,340 @@
+==================
+
+* Added support for the nRF BLE controller 0.2.0-4.prealpha. Includes drivers to access HCI, flash, clock control, and entropy hardware.
+  For details, see :ref:`nrfxlib:ble_controller_changelog`.
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Added the following samples:
+
+  * :ref:`central_bas` - demonstrates how do use the :ref:`bas_c_readme` to receive battery level information from a compatible device.
+  * :ref:`shell_bt_nus` - demonstrates how to use the :ref:`shell_bt_nus_readme` to receive shell commands from a remote device.
+
+* Added the following libraries:
+
+  * :ref:`bas_c_readme` - used to retrieve information about the battery level from a device.
+  * :ref:`shell_bt_nus_readme` - allows for sending shell commands from a host to the application.
+
+* Added :ref:`ble_console_readme` - a desktop application that can be used to communicate with an nRF device over *Bluetooth* Low Energy using the :ref:`shell_bt_nus_readme`.
+* Added Manufacturer Data filter to the :ref:`nrf_bt_scan_readme`.
+* Added application callbacks for the Output Report related operations in the HID service.
+
+
+Partition Manager
+-----------------
+
+* Partition Manager now handles all HEX file merging.
+* :ref:`ug_pm_static` of upgradable images is now supported.
+
+
+nRF Desktop
+===========
+
+* The nrf_desktop reference implementation is moved from the ``samples/`` folder to ``applications/``.
+* The nrf_desktop configuration channel now allows data to be exchanged between the device and host in both directions.
+
+
+Documentation
+=============
+
+* Added or updated documentation for the following samples:
+
+  * nRF9160:
+
+    * :ref:`secure_services`
+    * :ref:`secure_partition_manager`
+    * :ref:`aws_fota_sample`
+    * :ref:`lte_sensor_gateway`
+
+  * Bluetooth Low Energy:
+
+    * :ref:`central_bas`
+    * :ref:`bluetooth_central_hids`
+    * :ref:`peripheral_lbs`
+    * :ref:`shell_bt_nus`
+
+  * Other:
+
+    * :ref:`ppi_trace_sample`
+
+* Added or updated documentation for the following libraries:
+
+  * nRF9160:
+
+    * :ref:`lib_spm`
+    * :ref:`at_cmd_readme`
+    * :ref:`lib_download_client`
+    * :ref:`lib_aws_fota`
+    * :ref:`lib_fota_download`
+    * :ref:`lib_secure_services`
+
+  * Bluetooth Low Energy:
+
+    * :ref:`bas_c_readme`
+
+  * Other:
+
+    * :ref:`ppi_trace`
+    * :ref:`ble_console_readme`
+
+* Added or updated the following documentation:
+
+  * :ref:`ug_bt_ll_nrfxlib`
+  * :ref:`ug_multi_image`
+  * :ref:`partition_manager`
+  * :ref:`nrf_desktop`
+  * :ref:`shell_bt_nus_readme`
+
+* API documentation of all libraries now also mentions the location of header files and source files.
+
+Known issues
+************
+
+General
+=======
+
+* Flashing samples utilizing the Secure Partition Manager (SPM) might silently fail when using nrfjprog versions 10.2.0 and 10.1.1.
+  This issue is caused by ACL protection of flash memory and does not affect nrfjprog v9.8.1.
```

Comment:
"This issue is caused by ACL protection" , it's SPU , not ACL

Reply:
after discussing with @shantha-14 , it seems like we might be deleting this point tomorrow after all so I'll wait with any changes